cypress/e2e/shared/arduino.test.ts

```diff
@@ -0,0 +1,83 @@
+describe('Arduino onboarding', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+    cy.setFeatureFlags({firstMile: true, onboardArduino: true})
+  })
+
+  it('should load the main home-page that has the arduino tile', function() {
+    cy.getByTestID('home-page--header').should('exist')
```

Comment:
can this be moved down into the `describe('Arduino'` block?

Reply:
I was thinking we separate the homepage from the 'wizard'. The test name wasn't descriptive but to help make that distinction I renamed the test to 'Arduino Wizard' and it will contain all the tests that happen after the homepage tile is clicked. If this makes sense?